pubnub/pubnub_asyncio.py

```diff
@@ -566,6 +566,7 @@ def __init__(self, pubnub_instance):
         self.event_engine.get_dispatcher().set_pn(pubnub_instance)
         self.presence_engine.get_dispatcher().set_pn(pubnub_instance)
         self.loop = asyncio.new_event_loop()
+        self._heartbeat_periodic_callback = None
```

Comment:
Why did you have to set it to `None`?

Reply:
This should be initialized with `None` a long time ago. It's used [here](https://github.com/pubnub/python/blob/70533321960c97b933195d6427b84fa92976fc0d/pubnub/pubnub_asyncio.py#L490-L492) with a comparison to `None` and [here](https://github.com/pubnub/python/blob/master/pubnub/pubnub_asyncio.py#L494-L497) it is set with `_register_heartbeat_timer` method

Reply:
also it's now more similar to what's in `NativeSubscriptionManager` [here](https://github.com/pubnub/python/blob/70533321960c97b933195d6427b84fa92976fc0d/pubnub/pubnub.py#L165)